docker/util.sh

```diff
@@ -5,20 +5,42 @@
 #
 # Utility library to use in scripts
 
-apt_ver()
+# This function installs packages with specific versions and also pin them
+# (with prio 990) to those versions by appending pinning information to
+# /etc/apt/preferences.d/cachalot, so they are not upgraded accidentally.
+#
+# All packages should be specified with version: pkg=version.
+#
+# Version can have wildcards, see apt-get/apt_preferences documentation for
+# details
+apt_pin_install()
```

Comment:
This doesn't deserve separate commit?

Reply:
Yeah, maybe. Is true that pinning and apt wildcards are not the same, but OTOH if I separate it, I will have to put a normal `apt install` call in `docker/dlang` just to change it in the next commit.

Reply:
OK, done.